README.md

```diff
@@ -2,6 +2,9 @@
 ## Project setup
 
 ### Setup database:
+For this application was used [MySQL example employee database](https://dev.mysql.com/doc/employee/en/). In the default MySQL employees example database was changed some tables and added a few new tables.
+
+For setting up the database follow the next steps:
```

Comment:
### Setup database:
[MySQL example employee database](https://dev.mysql.com/doc/employee/en/) was used for this application, **but** with extra tables and fields.

To set up the database follow these steps: